icons/loader-circle.json

```diff
@@ -7,10 +7,21 @@
     "ericfennis"
   ],
   "tags": [
-    "load"
+    "loading",
+    "wait",
+    "busy",
+    "progress",
+    "spinner",
+    "spinning",
+    "throbber"
   ],
   "categories": [
+    "cursors",
     "multimedia",
-    "layout"
+    "layout",
+    "design"
+  ],
+  "aliases": [
+    "loader-2"
```

Comment:
I'm in renaming this one and adding an alias. But we can get in to some problems with renaming icons with `-2` on the end. If we add it as an alias we can never make a `loader-2` icon, because the name is already occupied by an alias. This is something we can think of if go to v1.

Reply:
@ericfennis Ok, so keep it as `loader-2` for now…?

Reply:
Yess unfortunately the best here.

Reply:
@ericfennis Ok, done.